packages/dds/tree/README.md

```diff
@@ -311,6 +311,10 @@ flowchart
             schema-view-->schema-stored
             dependency-tracking
             forest-->tree
+            undo-->change-family
+            undo-->rebase
+            undo-->repair
+            undo-->tree
```

Comment:
I forget if this graph is supposed to omit edges that implied by transitive dependencies. You may be able to determine that by comparing the edges listed here with the fence files.
If so, these last two edges can be removed.

Reply:
It looks like it is. I removed it cause it looks much nicer but I'm not sure if this counts as a transitive dependency since it's directly depending on some types from repair. I was looking through the dependencies to figure out the answer to that but it seems like this graph is very out of date so I'll just fix it in general and clarify in a different PR.